src/components/input.rs

```diff
@@ -196,9 +197,16 @@ pub(crate) fn BitcoinInput(mut update_var: Signal<String>, label: String, id: St
     }
 }
 
-/// Fee rate input validation component.
+/// Component to select the resolution transaction fee rate via input field or
+/// dropdown with fees fetched from Esplora and their expected confirmation targets.
 #[component]
-pub(crate) fn FeeRateInput(mut update_var: Signal<String>, label: String, id: String) -> Element {
+pub fn FeeRateSelector(
+    id: String,
+    label_input: String,
+    label_dropdown: String,
+    mut update_var: Signal<String>,
+    fee_estimates: Signal<Option<HashMap<u16, f64>>>,
```

Comment:
You can use this type alias: https://github.com/storopoli/scrow/blob/b21f93dab8523e24f8148d43cd9c48bc016e84e1/src/esplora.rs#L11-L12

```suggestion
    fee_estimates: Signal<Option<FeeEstimate>>,
```

Reply:
Done